packages/smooth_app/lib/data_models/food_preferences_model.dart

```diff
@@ -0,0 +1,209 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/cupertino.dart';
+import 'package:smooth_app/database/user_database.dart';
+import 'package:smooth_app/temp/user_preferences.dart';
+import 'package:http/http.dart' as http;
+
+import 'dart:developer';
+
+/* preferences JSON sample:
+
+https://world.openfoodfacts.org/api/v0/preferences
+
+[
+{
+id: "not_important",
+name: "Not important"
+},
+{
+id: "important",
+name: "Important"
+},
+{
+id: "very_important",
+name: "Very important"
+},
+{
+id: "mandatory",
+name: "Mandatory"
+}
+]
+
+*/
+
+Future<List<Preference>> fetchPreferences(http.Client client) async {
+  final response =
+  await client.get('https://world.openfoodfacts.org/api/v0/preferences');
```

Comment:
I suggest that we use an actual file for the init ("asset"?), in case there's no internet connection for the first app start.
Then we can refresh with translations, and possibly new values.

Reply:
Using the actual file for the initialization sounds very good.